zoo-css-challenge/index.html

```diff
@@ -8,6 +8,7 @@
  		<img src='https://sharp-villani-e65ba8.netlify.app/zoo.svg'
  			alt="zoo logo"
  			id="logo"
+			border=0 style="border:0; text-decoration:none; outline:none"
```

Comment:
```suggestion
```
It's not good practise to add CSS with the `style` attribute. We should add CSS in the `style.css` file instead